flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -127,13 +128,29 @@
     private boolean forceAvro = false;
     private long autoWatermarkInterval = 200;
 
+    // ---------- statebackend related configurations ------------------------------
     /**
      * Interval in milliseconds for sending latency tracking marks from the sources to the sinks.
      */
     private long latencyTrackingInterval = MetricOptions.LATENCY_INTERVAL.defaultValue();
 
     private boolean isLatencyTrackingConfigured = false;
 
+    /** Interval in milliseconds to perform periodic materialization. */
+    private long periodicMaterializeInterval =
+            StateBackendOptions.PERIODIC_MATERIALIZATION_INTERVAL.defaultValue();
```

Comment:
nit: can we add `Millis` or `Ms` to the variable name (as in `taskCancellationIntervalMillis`)?
ditto other added fields